server/cluster/cluster.go

```diff
@@ -1467,9 +1503,14 @@ func (c *RaftCluster) RegionWriteStats() map[uint64][]*statistics.HotPeerStat {
 	return c.hotStat.RegionStats(statistics.WriteFlow, c.GetOpts().GetHotRegionCacheHitsThreshold())
 }
 
-// CheckRWStatus checks the read/write status.
-func (c *RaftCluster) CheckRWStatus(region *core.RegionInfo) {
-	c.hotStat.CheckRWAsync(region)
+// CheckWriteStatus checks the write status.
+func (c *RaftCluster) CheckWriteStatus(peerInfo *core.PeerInfo) {
```

Comment:
It seems these two functions can be replaced by c.hotStat.xxx directly?

Reply:
removed